src/Components/CandidateTable/fakeData.ts

```diff
@@ -236,65 +214,40 @@ export const fakeUserData: User[] = [
 //
 
 export const fakeCandidatesDataJava: Candidate[] = [
-  // {
-  //   userID: 0,
-  //   hashName: "",
-  //   score: 0,
-  //   scoreProcent: 0,
-  //   rank: Rank.NM,
-  //   taskStartDate: "",
-  //   taskEndDate: "",
-  //   userLocation: "",
-  //   userLanguages: [],
-  //   userStatus: UserStatus.NotSelected,
-  // },
-
-  {
-    userID: 2,
-    hashName: "a7170cea",
+  {
+    hash: "fe646603542f1a01b70692b8c7a666d3d7933bf29ceea4d93b64687ba7170cea",
     score: 72,
-    scoreProcent: 10.8,
+    scorePercent: 10.8,
     rank: Rank.Junior,
-    taskStartDate: "2022.08.26 14:40",
-    taskEndDate: "2022.08.26 15:33",
-    userLocation: "United States, Ontario",
-    userLanguages: ["en"],
+    taskStartDate: new Date("2022-08-26 14:40"),
```

Comment:
BTW, our standard date format is as defined in verifiable credentials standard: https://github.com/doma-engineering/do-auth/blob/main/priv/ui/test/doauth_props.spec.ts#L52

The format is following ISO 8601 standard.

I would expect this format and convert it to JS Date early!

Don't fix this, I'll make an issue for another PR.